wannierberri/__tabulate.py

```diff
@@ -307,7 +307,7 @@ def plot_path_fat(self,
             Emax=E.max()+0.5
 
         klineall=[]
-        for ib in iband:
+        for ib in range(E.shape[1]):
```

Comment:
```suggestion
        for ib in range(len(iband)):
```

(Sorry, I know I suggested `range(E.shape[1])` but `range(len(iband))` seems more informative)